sdk/build.gradle

```diff
@@ -150,6 +149,10 @@ jacocoTestReport {
 task integrationTest(type: Test) {
     testClassesDirs = sourceSets.integrationTest.output.classesDirs
     classpath = sourceSets.integrationTest.runtimeClasspath
+
+    onlyIf {
+        System.getenv('OPERATOR_ID') != null && System.getenv('OPERATOR_KEY') != null
```

Comment:
Can we changed this to 
```groovy
        (System.getenv('OPERATOR_ID') != null && System.getenv('OPERATOR_KEY') != null)  || System.getenv('CONFIG_FILE') != null
````
since @Sean-Tedrow-LB and I usually run integration tests through a config file.

Reply:
Sure, done